root/etc/cont-init.d/13-nginx

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/with-contenv bash
+
+migrate(){
```

Comment:
Shouldn't it move to the new migration system?

Reply:
The way it's configured here, the logic will run on every init. I don't know how necessary that is, so we could consider making this use the new migration system, yes.

Off the top of my head I can't see why it would need to run more than once, but I'm not sure.

Reply:
Done in 050220e4b02a8f651f67bf6989fbfa5940075509